CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v2.2.8
+Fri 25 Jun 2021 11:36:14 BST
+
+* [c33a144f](https://github.com/hyperledger/fabric-sdk-node/commit/c33a144f) [FABN-1716](https://jira.hyperledger.org/browse/FABN-1716): Fix hang on application exit (#471)
+* [70b4cbc6](https://github.com/hyperledger/fabric-sdk-node/commit/70b4cbc6) [[FABN-1554](https://jira.hyperledger.org/browse/FABN-1554)] Add label support for HSM (#468) (#470)
+* [442999c1](https://github.com/hyperledger/fabric-sdk-node/commit/442999c1) [[FABN-1710](https://jira.hyperledger.org/browse/FABN-1710)] Fix HSM Persistence issues (#464) (#466)
```

Comment:
BTW, We've stopped using changelog.md in core Fabric since the github release page shows the commits. Up to you whether you want to continue or not...